source.txt

```diff
@@ -281,11 +281,10 @@ Table of Contents
        * 414 if the request URI is too long,
        * 416 if Range requests are supported by the server and the Range
              request can not be satisfied,
-       * 401 for all requests that don't have a bearer token with
-             sufficient permissions,
-       * 403 for all requests that either have insufficient scope, e.g. 
-             accessing a <module> for which no scope was obtained, or accessing
-             data outside the user's <storage_root>,
+       * 401 for all requests that don't have a valid bearer token,
```

Comment:
For which definition of "valid" though? Should perhaps be something like "for all requests with missing, expired or revoked token".